routes/auth-qr-code-png-route.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
This file is already in the _routes_ folder, do you think is necessary to add the _route_ suffix to the file name. The same criteria should applied to all the routes.